src/types/gecko-profile.js

```diff
@@ -173,6 +173,9 @@ export type GeckoProfile = {|
     stackwalk: number,
     toolkit: string,
     version: number,
+    // The appBuildID, sourceURL, physicalCPUs and logicalCPUs properties landed
+    // in Firefox 62, and are only optional because older processed profile
+    // versions may not have them. No upgrader was written for this change.
```

Comment:
No change needed, but I'd like to react on `No upgrader was written for this change`. Indeed even with an upgrader, you can't invent data you don't have, so you couldn't do much better :)